locales/fr/translation.json

```diff
@@ -144,6 +142,7 @@
   "Save": "Sauvegarder",
   "Sending...": "Envoi...",
   "Serial Number": "Numéro de série",
+  "Settings on day": "Paramètres sur {{day}}",
```

Comment:
Not consistent with the English version

![image](https://github.com/user-attachments/assets/400b6186-5e68-47cd-90ce-8b8c6ad0b9cb)